tools/subcategories.json

```diff
@@ -70,4 +71,4 @@
     "data-sources/oncall_user_group": "OnCall",
     "data-sources/synthetic_monitoring_probe": "Synthetic Monitoring",
     "data-sources/synthetic_monitoring_probes": "Synthetic Monitoring"
-}
+}
```

Comment:
Please configure your editor to add a new line at EOL and/or not removing the existing new line at EOL.

Reply:
Done, apologies!

Reply:
No worries!